sops/alerts/prometheus_job_down.adoc

```diff
@@ -0,0 +1,23 @@
+:toc:
+:toc-placement!:
+
+= ThreescalePrometheusJobDown
+
+toc::[]
+
+== Description
+
+* This alert willl trigger when a prometheus Job is Down, because `ServiceMonitor`/`PodMonitor` might have issues to scrape the application `Service`/`Pods`
+* It is a **really important alert**, because if you have alerts based on applicatin metrics that cannot be obtained because its associated prometheus job is failing, then you lose visibility of the application
```

Comment:
```suggestion
* It is a **CRITICAL ALERT**, because if you have alerts based on application metrics that cannot be obtained because its associated prometheus job is failing you will lose visibility of the application.
```